src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -156,16 +159,18 @@ public CaseDocumentAmController(DocumentManagementService documentManagementServ
     ) {
         validationUtils.validateDocumentId(documentId.toString());
 
-        ResponseEntity<StoredDocumentHalResource> documentMetadata =
+        Optional<StoredDocumentHalResource> documentMetadata =
```

Comment:
you can avoid duplicate statements by moving these two statements into a private method and return StoredDocumentHalResource from it.